docs/ref/release_notes.rst

```diff
@@ -0,0 +1,54 @@
+What's new in Iris-grib v0.13
+=============================
+
+:Release: 0.13.0
+:Date: T.B.D.
+
+Features
+--------
+
+* Added loading for data on Hybrid Height levels (surface type 118 in
+  code table 4.5).
```

Comment:
Assumedly we'll add GDT10 load/save support to this list as and when it arrives..?

Reply:
Absolutely, it should be incremental.
And hopefully, we'll all get in the habit of keeping it up-to-date.
I don't think we need an Iris-style |'contributions + aggregate' process.